### ISSUE TITLE
can't control with chrome, normal with firefox

### ISSUE THREAD

**Issue by extirpate** [joescii/lift-ng-demo#1]
git clone
changed the properties, sbt=0.13.8 --->  sbt=0.13.7
sbt
container:start

open with latest chrome 44.0.2403.155 m, can't control the slides with arrow and space keys.
open with firefox 40.0.2, everything is ok.

**Comment by joescii** [comment]
I'm surprised it works that way on Firefox.  Use PageUp/PageDown.  I made it this way because I often insert interactive components, causing me to advance when I use those otherwise common keys.  Also, I have found that usb clicker/pointers tend to map their slide advance buttons to PageUp/PageDown.

**Comment by extirpate** [comment]
is this not planned to advance by "UP DOWN LEFT RIGTH" keys?
I really want to know  the right way to use it , could you tell me how to ?

**Comment by joescii** [comment]
No, only PageUp/PageDown.  You can easily change this behavior by widening [this `if` statement](https://github.com/joescii/lift-ng-demo/blob/master/src/main/webapp/js/impress-presenter.js#L37).  The case block below it shows you the other key codes.

Also, if you are planning to adapt this to your own presentation, I recommend starting with the [Presentera giter8 template](https://github.com/joescii/presentera.g8).  This way the project will be blank and let you fill in your own details.

**Comment by extirpate** [comment]
really great work!
i can advance using presenter account , and forcing other screens' browser follow me.
but with different browser  this project seems not so compatible

**Comment by joescii** [comment]
Yeah, the cross-browser support is not very good.  The front-end work from [impress.js](https://github.com/impress/impress.js) was designed to run in only one browser, namely the presenter's.  Hence there is no concern for supporting browsers since the presenter would know to use a good one.
